### PR TITLE
Fixing style.scss so things can be overridden

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

File: style.scss [hunk 1]
```diff
@@ -39,68 +43,207 @@
...
+ * definitely should, so your CSS is as lean as possible, then only import what
+ * you want. The Core section is however mandatory.
+ *
+ * Any settings you find set in Scally that you do not wish to keep, simply
+ * redefine above/below the relevant `@import`. This means that if Scally, for
+ * example, sets your `$font-size` at `16px` and you wish it to be `14px`,
+ * simply redeclare it above the relevant `@import`, like so:
+ *
+   $font-size: 14;
+   @import "bower_components/scally/core/settings/typography";
```
Comment (dansimco) [inline style.scss:55]: Will referencing "bower_components" still work if the user uses a different bower directory name in their .bowerrc?

**Comment by AlecRaeside** [comment]
LGTM

File: style.scss (hunk 1 above)
Comment (chris-pearce) [inline style.scss:55]: Nope, see: https://github.com/westfieldlabs/scally/blob/style-scss/style.scss#L76-L78 comments

Comment (chris-pearce) [inline style.scss:55]: I wanted to put the path in a variable but you can't use variables in `@import` paths.

**Pull request merged**